Source/Scene/ImplicitTileset.js

```diff
@@ -246,10 +251,12 @@ function makeTileHeaderTemplate(tileJson) {
   }
 
   // content is handled separately, so remove content-related properties
-  delete template.content;
-
   if (hasExtension(template, "3DTILES_multiple_contents")) {
     delete template.extensions["3DTILES_multiple_contents"];
+  } else if (defined(template.contents)) {
+    delete template.contents;
+  } else if (defined(template.content)) {
+    delete template.content;
   }
```

Comment:
:thinking: this section can also be simplified I think.

Though the one thing to be careful of is make sure `template.extensions` is defined first, `delete template.foo` will work even if foo is undefined, but `delete template.foo.bar` will fail if foo is undefined.